library/src/main/java/com/woxthebox/draglistview/BoardView.java

```diff
@@ -534,7 +537,16 @@ public void setCustomDragItem(DragItem dragItem) {
     }
 
     public DragItemRecyclerView addColumnList(final DragItemAdapter adapter, final View header, boolean hasFixedItemSize) {
-        final DragItemRecyclerView recyclerView = new DragItemRecyclerView(getContext());
+        return addColumnList(adapter, header, hasFixedItemSize, null);
+    }
+
+    public DragItemRecyclerView addColumnList(final DragItemAdapter adapter, final View header, boolean hasFixedItemSize, @LayoutRes Integer recyclerViewLayoutId) {
```

Comment:
I rather not expose a new api with a layout res. Instead I propose to do the same as in DragListView, to inflate the xml here and set the scrollbars to false as default. Then you will be able to turn it on via code.